docs/source/models.rst

```diff
@@ -17,6 +17,23 @@ Text Detection
 --------------
 Localizing text elements in images
 
+.. list-table:: Text detection model zoo
+   :widths: 20 20 15 10 10 10
+   :header-rows: 1
+
+   * - Architecture
+     - Input shape
+     - # params
+     - Recall
```

Comment:
I think we also need to reference the test dataset on which we evaluate the models

Reply:
Indeed, I left the value blank but I assumed we would mention the test set used above the table since it will have to apply to all entries for easier comparison :+1: